src/elements/tabs/tab-nav-bar/tab-nav-bar.snapshot.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+
+import { fixture, testA11yTreeSnapshot } from '../../core/testing/private.ts';
+
+import type { SbbTabNavBarElement } from './tab-nav-bar.component.ts';
+import './tab-nav-bar.component.ts';
+
+describe(`sbb-tab-nav-bar`, () => {
+  describe('renders', () => {
+    let element: SbbTabNavBarElement;
+
+    beforeEach(async () => {
+      element = await fixture(
+        html`<sbb-tab-nav-bar>
+          <a href="javascript:void(0);" class="sbb-active">Nav item 1</a>
```

Comment:
would that work too? :)
```suggestion
          <a href="#" class="sbb-active">Nav item 1</a>
```